README.md

```diff
@@ -48,3 +59,66 @@ func main() {
 ```
 
 The function `Parse(s string)` is deprecated, and should no longer be used.
+
+# Global Privacy Platform v1.0
+
```

Comment:
Drop a link to the actual GPP spec, if someone wants to read though!

Reply:
Added.